torch/nn/functional.py

```diff
@@ -3113,6 +3119,9 @@ def multi_head_attention_forward(query,                  # type: Tensor
             be ignored by the attention.
         need_weights: output attn_output_weights.
         attn_mask: mask that prevents attention to certain positions.
+        use_chunk_proj_weight: use in_proj_weight insteady of q_proj_weight, k_proj_weight, v_proj_weight.
```

Comment:
Nit: "instead"

Also, this means that passing the flags for q, k, v weights is now the default? That's breaking backwards compatibility. Is that worth it?

Reply:
It preserves backwards compatiblity (use_chunk_proj_weight=True will use in_proj_weight, like before) but the name is confusing.

Reply:
I have added more docs for use_chunk_proj_weight